### PR TITLE
Added type checker (ts build)

### PR DESCRIPTION
**Pull request by AleksandrHorev**

### CODE REVIEW

**Review by BigRoma — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,8 +25,16 @@
         "url": "https://github.com/tonlabs/TON-Surf-Localization/issues"
     },
     "homepage": "https://github.com/tonlabs/TON-Surf-Localization#readme",
-    "dependencies": {
-        "@tonlabs/localization": "^2.3.0",
-        "typescript": "4.1.2"
+    "peerDependencies": {
+        "@react-native-clipboard/clipboard": "1.8.4",
+        "@tonlabs/uikit.localization": "2.3.2",
```
Comment (sertony) [inline package.json]: No need to have here

File: package.json [hunk 2]
```diff
@@ -25,8 +25,16 @@
         "url": "https://github.com/tonlabs/TON-Surf-Localization/issues"
     },
     "homepage": "https://github.com/tonlabs/TON-Surf-Localization#readme",
-    "dependencies": {
-        "@tonlabs/localization": "^2.3.0",
-        "typescript": "4.1.2"
+    "peerDependencies": {
+        "@react-native-clipboard/clipboard": "1.8.4",
```
Comment (sertony) [inline package.json]: No need to have here (it's not used in this module)

File: package.json [hunk 3]
```diff
@@ -25,8 +25,16 @@
...
+        "@tonlabs/uikit.localization": "2.3.2",
+        "react-native-localization": "2.1.7"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@types/react": "17.0.24",
+        "@tonlabs/uikit.localization": "2.3.2",
```
Comment (sertony) [inline package.json]: Need to revert it back
```suggestion
        "@tonlabs/localization": "^2.3.2",
```

File: package.json [hunk 4]
```diff
@@ -25,8 +25,14 @@
         "url": "https://github.com/tonlabs/TON-Surf-Localization/issues"
     },
     "homepage": "https://github.com/tonlabs/TON-Surf-Localization#readme",
-    "dependencies": {
-        "@tonlabs/localization": "^2.3.0",
-        "typescript": "4.1.2"
+    "peerDependencies": {
+        "react-native-localization": "2.1.7"
```
Comment (sertony) [inline package.json]:
```suggestion
        "react-native-localization": "^2.1.6"
```

File: package-lock.json [hunk 5]
```diff
@@ -4,29 +4,86 @@
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
-        "@tonlabs/localization": {
+        "@tonlabs/uikit.localization": {
```
Comment (sertony) [inline package-lock.json]: Besides, need to update package-lock

**Review by sertony — APPROVED** [review]

**Pull request merged**